content/v2.0/monitor-alert/send-email.md

```diff
@@ -0,0 +1,235 @@
+3. In the **Name** field, enter a descriptive name, for example, **Send alert email**, and then enter how often to run the task in the **Every** field, for example, `10m`. For more detail, such as using cron syntax or including an offset, see [Task configuration options](/v2.0/process-data/task-options/).
+
+4. In the right panel, enter the following detail in your **task script** (see [examples below](#examples)):
+   - Import the [Flux HTTP package](/v2.0/reference/flux/stdlib/http/).
+   - (Optional) Store your API key as a secret for reuse. First, [add your API key as a secret](/v2.0/security/secrets/manage-secrets/add/), and then import the [Flux InfluxDB Secrets package](/v2.0/reference/flux/stdlib/secrets/).
+   - Query the `statuses` measurement in the `_monitoring` bucket to retrieve all statuses generated by your check.
+   - Set the time range to monitor; use the same interval that the task is scheduled to run. For example, `range (start: -task.every)`.
+   - Set the `_level` to alert on, for example, `crit`, `warn`, `info`, or `ok`.
+   - Use the `map()` function to evaluate the criteria to send an alert using `http.post()`.
+   - Specify your email service `url` (endpoint), include applicable request `headers`, and verify your request `data` format follows the format specified for your email service.
+
+#### Examples
+
+{{< tabs-wrapper >}}
+{{% tabs %}}
+[SendGrid](#)
+[AWS SES](#)
+[Mailjet](#)
+[Mailgun](#)
+{{% /tabs %}}
+
+<!-------------------------------- BEGIN SendGrid -------------------------------->
+{{% tab-content %}}
+
+The example below uses the SendGrid API to send an alert email when more than 3 critical statuses occur since the previous task run.
+
+```js
+import "http"
+
+// Import the Secrets package if you store your API key as a secret.
+// For detail on how to do this, see Step 4 above.
+import "influxdata/influxdb/secrets"
+
+// Retrieve the secret if applicable. Otherwise, skip this line
+// and add the API key as the Bearer token in the Authorization header.
+SENDGRID_APIKEY = secrets.get(key: "SENDGRID_APIKEY")
+
+numberOfCrits = from(bucket: "_monitoring")
+	|> range(start: -task.every)
+	|> filter(fn: (r) => r.measurement == "statuses" and r.level == "crit")
+	|> count()
+
+numberOfCrits
+  |> map(fn: (r) => (if r._value > 3 then {
+    r with _value: http.post(
+      url: "https://api.sendgrid.com/v3/mail/send",
+        headers: {"Content-Type": "application/json", Authorization: "Bearer ${SENDGRID_APIKEY}",
+        data: bytes(v: "{
+          \"personalizations\": [{
+            \"to\": [{
+            \"email\": \”jane.doe@example.com\"}],
+            \"subject\": \”InfluxData critical alert\"
+            }],
+            \"from\": {\"email\": \"john.doe@example.com\"},
+            \"content\": [{
+              \"type\": \"text/plain\",
+              \"value\": \”Example alert text\"
+            }]
+        }\""))} else {r with _value: 0}))
+```
```

Comment:
I know I'm ridiculous 😄, but `headers` and `data` should be indented the same as `url`.

Reply:
😄 I love this trait about you.